PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh

```diff
@@ -216,10 +216,13 @@ case $DistortionCorrection in
         # -- Spin Echo Field Maps --  
         # --------------------------
 
-        if [[ ${SEUnwarpDir} = "x" || ${SEUnwarpDir} = "y" ]] ; then
-          ScoutInputName="${SpinEchoPhaseEncodePositive}"
-        elif [[ ${SEUnwarpDir} = "-x" || ${SEUnwarpDir} = "-y" || ${SEUnwarpDir} = "x-" || ${SEUnwarpDir} = "y-" ]] ; then
-          ScoutInputName="${SpinEchoPhaseEncodeNegative}"
+        if [[ ${SEUnwarpDir} = [xyij] ]] ; then
```

Comment:
This looks like it will still error on z/k.

Reply:
I'll fix this once @glasserm agrees that 180fbe5 is ok.

Reply:
This stays "as is", given that Jesper has confirmed that 'topup' indeed doesn't allow z as the PE direction.